projects/igniteui-angular/src/lib/grids/filtering/excel-style/excel-style-filtering.component.ts

```diff
@@ -793,4 +802,13 @@ export class IgxGridExcelStyleFilteringComponent extends BaseFilteringComponent
 
         return value;
     }
+
+    private checkSortingDirection(items: any[]): boolean {
```

Comment:
'checkSortingDirection' name does not imply that it would return true when the direction is descending. We need a more appropriate name.